src/openrct2/ride/RideRatings.cpp

```diff
@@ -90,9 +94,27 @@ static void ride_ratings_begin_proximity_loop(RideRatingUpdateState& state);
 static void RideRatingsCalculate(RideRatingUpdateState& state, Ride& ride);
 static void RideRatingsCalculateValue(Ride& ride);
 static void ride_ratings_score_close_proximity(RideRatingUpdateState& state, TileElement* inputTileElement);
-
 static void ride_ratings_add(RatingTuple* rating, int32_t excitement, int32_t intensity, int32_t nausea);
 
+RideRatingUpdateStates& RideRatingGetUpdateStates()
+{
+    return gRideRatingUpdateStates;
+}
+
+void RideRatingResetUpdateStates()
+{
+    RideRatingUpdateState nullState{};
+    nullState.State = RIDE_RATINGS_STATE_FIND_NEXT_RIDE;
+
+    std::fill(gRideRatingUpdateStates.begin(), gRideRatingUpdateStates.end(), nullState);
+}
+
+RideRatingUpdateState& RideRatingGetUpdateState(size_t index)
```

Comment:
```suggestion
static RideRatingUpdateState& RideRatingGetUpdateState(size_t index)
```

Reply:
I actually forgot about this, it's no longer used, this was to keep each commit in a working state, ill remove it.